### PR TITLE
202-publishing happens before the cypress tests finish

### LINKED ISSUE #202: Publishing Happens before Tests Finish
#### Describe the bug
Cypress tests are ran after publishing and tagging of packages.

#### What is the expected behavior?
Package and tag steps should wait for the cypress tests job to finish before executing. 

#### What are the steps to reproduce?
1.  Go observe our latest master merge: https://app.circleci.com/pipelines/github/brightlayer-ui/angular-workflows/184/workflows/4ed0532e-f96a-4451-8271-f63e0ecfa3a0

#### Screenshots / Screen recording
image

### PR DESCRIPTION
**Pull request by raghavenc5**
Fixes #202  .

#### Changes proposed in this Pull Request:
- 
- 
- 

#### Screenshots / Screen Recording (if applicable)
-

#### To Test:
- 

#### Any specific feedback you are looking for?
-

### CODE REVIEW

**Comment by raghavenc5** [comment]
@joebochill Kindly check the modified workflow steps and let me know if any changes required.

**Review by JeffGreiner-eaton — APPROVED** [review]

**Comment by JeffGreiner-eaton** [comment]
just need to resolve the conflicts

**Comment by raghavenc5** [comment]
> just need to resolve the conflicts

@JeffGreiner-eaton  Resolved the conflicts

File: login-workflow/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 ### Fixed
 
+-   Fixed Publishing Happens before Tests Finish in the circleci config file ([#202](https://github.com/etn-ccis/blui-angular-workflows/issues/202)).
```
Comment (huayunh) [inline login-workflow/CHANGELOG.md:7]: Please remove this from the changelog. This is not a change happened to the workflow itself; our users does not need to know about this change.

**Comment by huayunh** [comment]
and while you are here, can you run `yarn prettier`? Your previous CI/CD check has failed because of it

**Review by huayunh — APPROVED** [review]

**Comment by raghavenc5** [comment]
> and while you are here, can you run `yarn prettier`? Your previous CI/CD check has failed because of it

@huayunh Updated prettier changes

**Pull request merged**